source/extensions/common/wasm/wavm/wavm.cc

```diff
@@ -437,8 +479,8 @@ void getFunctionWavmReturn(WasmVm* vm, absl::string_view functionName,
     throw WasmVmException(fmt::format("Bad function signature for: {}", functionName));
   }
   *function = [wavm, f](Context* context, Args... args) -> R {
-    CALL_WITH_CONTEXT_RETURN(invokeFunctionChecked(wavm->context(), f, {args...}), context,
-                             uint32_t, i32);
+    CALL_WITH_CONTEXT_RETURN(invokeFunctionChecked(wavm->context_, f, {args...}), context, uint32_t,
```

Comment:
Sorry for not review your stacked PR on time!

Reply:
np, I'll add a tag next time e.g. (stacked)